classes/requests/helpers/class-truelayer-helper-signing.php

```diff
@@ -9,11 +9,11 @@
 	exit;
 }
 
-use TrueLayer\Signing\Signer;
-use Ramsey\Uuid\Uuid;
+use KrokedilTrueLayerDeps\TrueLayer\Signing\Signer;
+use KrokedilTrueLayerDeps\Ramsey\Uuid\Uuid;
 
 /**
- * Heler class for the request signing.
+ * Helper class for the request signing.
```

Comment:
`get_tl_signature()` shouldn't be needed anymore and you can drop the direct dependency on the signing package too 👍